src/components/button/action/style/button.scss

```diff
@@ -0,0 +1,6 @@
+
+[data-focus="button-action"] {
```

Comment:
rename it `button`, don't forget to rename it also on the `div` in the component itself